AnnoyancesFilter/Other/sections/annoyances.txt

```diff
@@ -30,6 +30,8 @@ instagram.com##._aagw:empty
 !
 ! End: Instagram.com
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/157914
+gismeteo.ru#%#AG_onLoad(function(){let e=document.querySelectorAll('label[data-title="Объясняем.рф"]'),l=[];for(let t=0;t<e.length;t++)l.push(e[t].htmlFor);let r=l.map(e=>document.querySelector(`div[data-id="${e}"`));if(e||r){for(let o=0;o<e.length;o++)e[o].remove();r.map(e=>e.remove())}let a=document.querySelectorAll(".tab-labels > label:first-child");a&&setTimeout(()=>{for(let e=0;e<a.length;e++)a[e].click(),console.log("click")},1e3)});
```

Comment:
Could you rewrite name of the variables to make them more clear?
```javascript
let e = document.querySelectorAll('label[data-title="Объясняем.рф"]'),
for (let e = 0; e < a.length; e++)
```
Seems `let e` are two different variables, doesn't it?

```javascript
r=l.map(e=>document.querySelector(`div[data-id="${e}"`)
```
Should not there be `]` after `div[data-id="${e}"`?

```javascript
console.log("click")}
```
I guess it's not needed.

Reply:
Btw, I guess we can limit the rule only for mobile platforms. I don't see those elements on desktop.

Reply:
```javascript
AG_onLoad(function() {
    const labels = document.querySelectorAll('label[data-title="Объясняем.рф"]');
    let dataIds = [];
    for (let l = 0; l < labels.length; l++) dataIds.push(labels[l].htmlFor);
    let divs = dataIds.map((l => document.querySelector(`div[data-id="${l}"]`)));
    if (labels || divs) {
        for (let l = 0; l < labels.length; l++) labels[l].remove();
        divs.map((l => l.remove()))
    }
    const activeItems = document.querySelectorAll(".tab-labels > label:first-child");
    activeItems && setTimeout((() => {
        for (let l = 0; l < activeItems.length; l++) activeItems[l].click()
    }), 1e3);
})```